pkg/custom_detectors/custom_detectors.go

```diff
@@ -99,6 +99,10 @@ func (c *customRegexWebhook) FromData(ctx context.Context, verify bool, data []b
 	close(resultsCh)
 
 	for result := range resultsCh {
+		// NOTE: I don't believe this is being set anywhere else, hence the map assignment.
+		result.ExtraData = map[string]string{
```

Comment:
for the reviewer: could check for `nil` here but it doesn't look as nice. dont have a strong opinion on this one.